packaging/aznfs/DEBIAN/postinst

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Exit on error.
+set -e
+
+echo "===== postinst called. ====="
```

Comment:
we would want to remove these echos from all these files as they would not look good when user is installing